README.md

```diff
@@ -22,6 +22,26 @@ A graphical editor for commutative diagrams that exports to tikz-cd.
 - A history system with undo and redo.
 - Panning.
 
+## Building
+Make sure you have [installed yarn](https://yarnpkg.com/lang/en/docs/install/).
+
+Clone the repository. Run:
+```
+make
+```
+Open `src/index.html` in your favourite web browser.
+
+If that doesn't work, [open an
+issue](https://github.com/varkor/quiver/issues/new) detailing the problem.
+
+<blockquote>
```

Comment:
How common a problem is this?

Reply:
This appears to be a very specific problem with an alternative `yarn` command on Unix. I would be inclined to remove this, assuming that users are able to search for solutions to this error themselves.

Reply:
It affects Debian stable, Ubuntu (except the latest version), and derivatives of either (e.g. Linux Mint). So most distros that use apt for package management.